server/index.js

```diff
@@ -13,9 +13,32 @@ app.get('/items/:itemId/reviews', (req, res) => {
       console.log('error getting item reviews: ', err);
       res.status(400).send(err);
     } else {
-      var parsedData = JSON.parse(data)
       console.log('success getting reviews for item');
-      res.status(200).send(parsedData);
+      res.status(200).send(data);
+    }
+  });
+});
+
+app.get('/reviews/:reviewId', (req, res) => {
+  db.getInfoForReview(1, (err, data) => {
```

Comment:
Is this going to be 1 or the id sent from the client?

Reply:
this will be the id from the client

Reply:
> Everything looks fine, I'm just wondering if there's a better way to select everything you want from the database

what would you suggest?
I'm going to merge this so I can move forward, but if you have a suggestion I'd be happy to hear about it and possibly refactor.

Reply:
I don't know why that got commented on this thread specifically